src/main/resources/liquibase/1.0/changelog-1.0.3.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:

```

Comment:
we don't deploy this app, you can remove this file and write this update in 1.0.0